theme/tableediting.css

```diff
@@ -8,3 +8,10 @@
  * it acts as a message to the builder telling that it should look for the corresponding styles
  * **in the theme** when compiling the editor.
  */
+
+.ck-content .table table {
```

Comment:
Please excuse my 2018 friend :) I didn't validate this when working on it - just changed the styles. I think tha this also should land in `TableSelection` and not in editing part, am I right?

Reply:
I'm not going to move this to theme-lark :cry: I would like to avoid `table-selection` branch handling on differnet repos. I'll add this remark to `UI/UI` issue though.